Formula/b2sum.rb

```diff
@@ -0,0 +1,20 @@
+class B2sum < Formula
+  desc "BLAKE2 b2sum reference binary"
+  homepage "https://github.com/BLAKE2/BLAKE2"
+  url "https://github.com/BLAKE2/BLAKE2/archive/20160619.tar.gz"
+  sha256 "cbac833ccae56b5c6173dbeaf871aa99b32745cf7a994c7451d4533ecda55633"
+
+  def install
+    cd "b2sum" do
+      # https://github.com/BLAKE2/BLAKE2/issues/25
+      system "make", "NO_OPENMP=1"
+      bin.install "b2sum"
+      man1.install "b2sum.1"
```

Comment:
There is a makefile install target, but it installs to /usr/local/bin.

Not sure why using the makefile install target is important since there is only a single compilation output (b2sum) and a single pre-generated man.1 file.

See : https://github.com/BLAKE2/BLAKE2/blob/master/b2sum/makefile

Do you have any example of a formula making use of a similar makefile install target?